src/Raven.Studio/typescript/models/database/index/configurationItem.ts

```diff
@@ -3,13 +3,16 @@ import jsonUtil = require("common/jsonUtil");
 import configuration = require("configuration");
 
 class configurationItem {
-    static readonly ConfigurationOptions = [
-        configuration.indexing.maxTimeForDocumentTransactionToRemainOpen,
-        configuration.indexing.minNumberOfMapAttemptsAfterWhichBatchWillBeCanceledIfRunningLowOnMemory,
-        configuration.indexing.mapTimeout,
-        configuration.indexing.mapTimeoutAfterEtagReached
+
+    static readonly ServerWideIndexingConfigurationOptions = [        
+        configuration.indexing.size,
+        configuration.indexing.cleanupInterval,
+        configuration.indexing.maxTimeToWaitAfterFlushAndSyncWhenExceedingScratchSpaceLimit
     ];
 
+    static readonly PerDatabaseIndexingConfigurationOptions: Array<string> = _.sortBy(_.pullAll(_.values<string>(configuration.indexing), 
```

Comment:
do you know that pullAll mutates input array? Is it desired here?

Reply:
1. Here it doesn't matter that the input array is mutated -
    because it is the array resulting from `_.values()`, Not the original list.
2. I can use `_.difference` if you prefer, it will be same result

Reply:
ok you are right we have values there as well, so it is fine.